examples/cpp/configuration/PublisherApp.cpp

```diff
@@ -17,8 +17,6 @@
  *
  */
 
-#include "PublisherApp.hpp"
```

Comment:
Header files referring to a direct header should be kept first. Please, revert this change